deployment_cfn/.specific/deployment_options.adoc

```diff
@@ -1,8 +1,8 @@
-// Edit this placeholder text to accurately describe your architecture.
+// Edit this placeholder text as necessary to describe the deployment options.
 
-This Quick Start provides two deployment options:
+This Quick Start provides the following deployment options:
```

Comment:
```suggestion
This Quick Start provides the following deployment options. These deploy links open the AWS CloudFormation console with the template URL populated.
```
```suggestion
This Quick Start provides the following deployment options:
```

Reply:
I'd add this sentence (above) and remove the similar note from the `deployment_steps_cfn.adoc`.

Same suggestion applies for the EKS `deployment_options.adoc` and `deployment_steps_eks.adoc`.